WoW-Realm-Tracker/Models/Realm.swift

```diff
@@ -1,27 +1,36 @@
 import Runes
 import Argo
+import Curry
 
 struct Realm {
-    let name: String
-    let type: String
-    let status: Bool
+    enum RealmType: String, CustomStringConvertible {
+        case PvP = "pvp"
```

Comment:
Any reason for the string value to not be "PvP" and etc for the others? That way you can call `.rawValue` on the enum to get that representation automatically.